ibis/backends/base/sql/alchemy/query_builder.py

```diff
@@ -144,6 +144,13 @@ def _format_table(self, op):
         elif ctx.is_extracted(op):
             if isinstance(orig_op, ops.SelfReference):
                 result = ctx.get_ref(op)
+            elif isinstance(alias, str):
+                result = sa.Table(
+                    alias,
+                    sa.MetaData(),
+                    *translator._schema_to_sqlalchemy_columns(orig_op.schema),
+                    quote=translator._quote_table_names,
+                )
```

Comment:
Just a thought, but can we avoid the heavier-weight `sa.Table` and using `sa.table(alias, *translator...)`?

That said, take it with a grain of salt since this entire thing is going away soon.